src/apps/Agreement/DisputableApps/DisputableApps.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import {
+  DataView,
+  ContextMenu,
+  ContextMenuItem,
+  useTheme,
+  GU,
+  useLayout,
+  Help,
+  TokenBadge,
+} from '@aragon/ui'
+import { KnownAppBadge } from '../../../templates/kit'
+import InfoField from './../InfoField'
+import { AppItemsType } from './../prop-types'
+import SubtleLabel from './SubtleLabel'
+import TokenAmount from './TokenAmount'
+
+const EXPANDABLE_ROW_GAP = `${3 * GU}px`
+
+const DisputableApps = React.memo(({ items }) => {
+  const { layoutName } = useLayout()
+  const theme = useTheme()
+
+  return (
+    <DataView
+      fields={[
+        { label: 'App', priority: 1, childStart: true },
+        { label: 'Actions', align: 'left' },
+      ]}
+      entries={items}
+      renderEntry={entry => renderEntry(entry, layoutName)}
```

Comment:
Might be better to use `compactMode` instead, since we normally group them in just two categories (rather than "small", "medium", and "large")